backend/src/app/media.clj

```diff
@@ -213,7 +213,7 @@
 ;; --- Fonts Generation
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
-(def all-fotmats #{"font/woff2", "font/woff", "font/otf", "font/ttf"})
+(def all-formats #{"font/woff2", "font/woff", "font/otf", "font/ttf"})
```

Comment:
?

Reply:
I will remove it in other commit. This is an unused var.